ci/steps/notify.py

```diff
@@ -315,3 +316,9 @@ def send_notification(
                 channel=cicd_cfg.notify.slack_channel,
                 title='Build Logs',
             )
+
+    # check if we have a status file indicating that some previous step failed, exit with error then
+    with open(os.path.join(repo_dir, 'status.txt')) as f:
```

Comment:
this will fail if no such file exists - is this the intended semantics?

Reply:
yes, it must exist, otherwise Tekton does not even trigger the task.